src/resource_clients/actor.ts

```diff
@@ -57,7 +57,6 @@ export class ActorClient extends ResourceClient {
      * https://docs.apify.com/api/v2#/reference/actors/run-collection/run-actor
      */
     async start(input?: unknown, options: ActorStartOptions = {}): Promise<ActorRun> {
-        // input can be anything, pointless to validate
```

Comment:
> The API will accept whatever you set the content-type to. E.g. if you set it to application/pdf then it will process it as a buffer.

would be good to note this in the comment instead of removing it, same for line 108